File: examples-sdk/python/604-db-elastic-search.py
#!/usr/bin/env python3
"""604-db-elastic-search — Python SDK equivalent
Compile: vil compile --from python --input 604-db-elastic-search.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/604-db-elastic-search.ts
#!/usr/bin/env tsx
// 604-db-elastic-search — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 604-db-elastic-search.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
